vendor/golang.org/x/crypto/ssh/certs.go

```diff
@@ -320,7 +320,7 @@ func (c *CertChecker) Authenticate(conn ConnMetadata, pubKey PublicKey) (*Permis
 // the signature of the certificate.
 func (c *CertChecker) CheckCert(principal string, cert *Certificate) error {
 	if c.IsRevoked != nil && c.IsRevoked(cert) {
-		return fmt.Errorf("ssh: certicate serial %d revoked", cert.Serial)
```

Comment:
I would prefer not to make any changes to vendored code, even if it does fix a typo.